### PR TITLE
Edit to remove optional fields of contacts

### LINKED ISSUE #202: [PE-D] Optional Email Field
Since email is optional, it should be allowed for empty edit e/ to pass to remove just the email field. However this command is not allowed. In the case whereby I simply want to remove email, I am unable to do so.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: habi39/ped#8

### LINKED ISSUE #261: Edit to remove optional phone field
Related to #202

### LINKED ISSUE #262: Edit to remove optional address field
Related to #202

### PR DESCRIPTION
**Pull request by edelinetenges**
Resolves #202 
Resolves #261 
Resolves #262

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
             + "provided is invalid";
     public static final String MESSAGE_MORE_THAN_ONE_SAME_PREFIX = "There should only be 1 option";
     public static final String MESSAGE_DUPLICATE_APPOINTMENT =
-            "You already have an appointment with the same name/date in the address book.";
+            "You already have an appointment with the same name/date/address in the address book.";
```
Comment (clarlzx) [inline src/main/java/seedu/address/commons/core/Messages.java:18]: It's an "and" relationship if I remembered correctly, so maybe "with the exact same name, date and address" would be better?

**Review by clarlzx — APPROVED** [review]
Looks good to be merged! 😄

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 1 above)
Comment (edelinetenges) [reply src/main/java/seedu/address/commons/core/Messages.java:18]: Okay! I'll update it accordingly.

Comment (clarlzx) [reply src/main/java/seedu/address/commons/core/Messages.java]: Okay 😄 👍

**Review by Stratostorm — APPROVED** [review]
Good to merge

**Pull request merged**